src/main/java/graphql/parser/GraphqlAntlrToLanguage.java

```diff
@@ -132,7 +140,7 @@ public Void visitOperationDefinition(GraphqlParser.OperationDefinitionContext ct
         } else if (operationTypeContext.getText().equals("mutation")) {
             return OperationDefinition.Operation.MUTATION;
         } else {
-            throw new RuntimeException();
+            throw new RuntimeException("InternalError: unknown operationTypeContext="+operationTypeContext.getText());
```

Comment:
This was a helpful message when I was debugging... but it does make me wonder if the grammar should be handling the mutation and query keywords. Now that 'name' is a rule rather than a token, we should be able to use the mutation and query keywords to the grammar.